extensions/wikia/EditPreview/js/editpage.event.helper.js

```diff
@@ -90,10 +91,50 @@ define('editpage.event.helper', ['wikia.window'], function(window, ace){
 		return $('#categories');
 	}
 
+	function IFrameForm(url) {
+		if (!this instanceof IFrameForm) {
+			return new IFrameForm(url);
+		}
+
+		this.time = new Date().getTime();
+		this.form = $('<form></form>')
+			.attr('action', url)
+			.attr('style', 'display:none')
+			.attr('target', 'iframe' + this.time)
+			.attr('method', 'post')
+			.attr('id', 'form' + this.time)
+			.attr('name', 'form' + this.time);
+
+		this.addParameter = function (parameter, value) {
+			$("<input type='hidden' />")
+				.attr('name', parameter)
+				.attr('value', value)
+				.appendTo(this.form);
+		};
+
+		this.send = function (frameRoot, callback) {
+			var iframe = $('<iframe></iframe>')
```

Comment:
is there any reason for creating `iframe` on just before send or could it be create while instantiating `IFrameForm`?